commands/alpha/rpkg/clone/command.go

```diff
@@ -200,6 +204,24 @@ func (r *runner) runE(cmd *cobra.Command, args []string) error {
 	return nil
 }
 
+func (r *runner) packageAlreadyExists(packageName string) error {
```

Comment:
Nit: This looks to be copied-and-pasted code, other than the error message?  Maybe put in a shared package, and return `(bool, error)`, where error => an error in fetching, and true/false => package found/not-found.

Reply:
done! Thanks